wdio/specs/contacts.js

```diff
@@ -45,5 +45,25 @@ describe('Municipality contacts page', () => {
     browser.logout();
   });
 
+  it('should be able to see contact info icons', () => {
```

Comment:
👍 
Note that now days, these kind of tests I prefer to do with Elm unit tests

Reply:
Noted.